builder/yandex/config.go

```diff
@@ -274,6 +279,10 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 			errs, errors.New("a folder_id must be specified"))
 	}
 
+	if c.TargetImageFolderID == "" {
```

Comment:
Right above this check there is validation for `FolderID` if both `source_image_folder_id` and `target_image_folder_id` are set does the user need to specify a value for `folder_id` as well? 

If not, maybe this condition can be moved above https://github.com/hashicorp/packer/pull/9080/files#diff-28716c25290f36af6513f08db37616b4R277

Reply:
Yeah, if both `source_image_folder_idz and `target_image_folder_id` are set value of `folder_id` is still required and used to set folder where provision a temporary Virtual Machine in.